packages/sales-api-service/src/server/routes/__tests__/staging-exceptions.spec.js

```diff
@@ -37,21 +37,65 @@ describe('staging exceptions handler', () => {
       expect(JSON.parse(result.payload)).toMatchObject({ stagingException })
     })
 
-    it('adds a transaction file exception if the payload contains a transactionFileException object', async () => {
-      const transactionFileException = {
-        name: 'string',
-        description: 'string',
-        json: 'string',
-        notes: 'string',
-        type: 'Failure',
-        transactionFile: 'string',
-        permissionId: 'string'
-      }
-      createTransactionFileException.mockResolvedValueOnce(transactionFileException)
-      const result = await server.inject({ method: 'POST', url: '/stagingExceptions', payload: { transactionFileException } })
-      expect(createTransactionFileException).toHaveBeenCalledWith(transactionFileException)
-      expect(result.statusCode).toBe(200)
-      expect(JSON.parse(result.payload)).toMatchObject({ transactionFileException })
+    describe('if the payload contains a transactionFileException object', () => {
+      let transactionFileException
+      beforeEach(() => {
+        transactionFileException = {
+          name: 'string',
+          description: '{ "json": "string" }',
+          json: 'string',
+          notes: 'string',
+          type: 'Failure',
+          transactionFile: 'string',
+          permissionId: 'string'
+        }
+        createTransactionFileException.mockResolvedValueOnce(transactionFileException)
+        createDataValidationError.mockResolvedValue()
+      })
+
+      it('adds a transaction file exception', async () => {
+        const result = await server.inject({ method: 'POST', url: '/stagingExceptions', payload: { transactionFileException } })
+        expect(createTransactionFileException).toHaveBeenCalledWith(transactionFileException)
```

Comment:
The beforeEach for the describe block needs to be clearing out your mocks: `beforeEach(jest.clearAllMocks)`, otherwise this could be a false positive. Also, should really be multiple tests rather than multiple expect statements here. And the last expect should be a snapshot test.